Flat/src/PandaAnalyzer.cc

```diff
@@ -290,7 +293,17 @@ double PandaAnalyzer::GetCorr(CorrectionType ct, double x, double y)
     return 1;
   }
 }
-
+double PandaAnalyzer::GetError(CorrectionType ct, double x, double y) {
+  if (h1Corrs[ct]!=0) {
+    return h1Corrs[ct]->Error(x); 
+  } else if (h2Corrs[ct]!=0) {
+    return h2Corrs[ct]->Error(x,y);
+  } else {
+    PError("PandaAnalyzer::GetCorr",
+       TString::Format("No correction is defined for CorrectionType=%u",ct));
```

Comment:
This error message should be fixed. First argument should be `"PandaAnalyzer::GetError"` and second should specify `No error` (there could be a `TF1` correction defined at this index, which carries no error and should throw this error).

Reply:
Agree. looks like I copied and pasted Gui's copy-and-pasting.